CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.0.4]
+### Fixed
+- Changing Newtonsoft.Json to JsonUtility

```

Comment:
Please explain the issue, as `Changing Newtonsoft.Json to JsonUtility` is the solution that caused the issue to be fixed.

Reply:
done